content/direct-deposit-management/release-notes/2024-12-11.md

```diff
@@ -0,0 +1,3 @@
+Dependents are no longer allowed to make updates to direct deposit information.  This change is implemented to prevent potential data conflicts with downstream systems.
```

Comment:
```suggestion
Dependents are no longer allowed to make updates to direct deposit information. This change is implemented to prevent potential data conflicts with downstream systems.
```